draft-ietf-rtcweb-jsep.xml

```diff
@@ -513,10 +513,13 @@ setRemote(ANSWER) |                                   |
           <t>Note that gathering phases only gather the candidates
           needed by new/recycled/restarting m= sections; other m=
           sections continue to use their existing candidates. Also,
-          when bundling is active, candidates are only gathered (and
-          exchanged) for the m= sections referenced in BUNDLE-tags, as
-          described in
-          <xref target="I-D.ietf-mmusic-sdp-bundle-negotiation" />.</t>
+          if an m= section is bundled (either by having had
```

Comment:
this parenthetical is somewhat hard to parse. Perhaps "either by a successful bundle negotiation or by being marked as bundle-only"